pkg/monitor/cluster/cluster.go

```diff
@@ -125,7 +132,6 @@ func (mon *Monitor) Monitor(ctx context.Context) (errs []error) {
 		mon.emitPodConditions,
 		mon.emitReplicasetStatuses,
 		mon.emitStatefulsetStatuses,
-		mon.emitSummary,
```

Comment:
Don't delete this. Just remove the provisioning state. 
The main idea of this is that this is an aggregated data metric. We have the same data in the other metrics but it is very hard to expose them as a single table for exports.

Reply:
@mjudeikis

1. For what purpose do we need this export?
2. Is it ok for that purpose that the export is going to be incomplete (because we run this function only if API server is healthy)?

Reply:
Reporting on the SD side to have some visibility in average cluster sizes, locations. It will need to be extended later too. But this is out of scope here.  And yes, this is fine if API server is not healthy.

Reply:
@mjudeikis ok, I'll revert. But how do we consume this metric? I didn't manage to find where we use it (I do not see any alerts or dashbaords) and decided to clean this up.

Reply:
Ok, I think I found the dashbaord. It was hiding in a separate folder away from our every day dashbaords.